cosmoz-default-tree.html

```diff
@@ -23,134 +32,125 @@
 
 	Cosmoz.DefaultTree.prototype = Object.create(Cosmoz.Tree.prototype);
 
-	Cosmoz.DefaultTree.prototype.getNodeByProperty = function (propertyName, propertyValue) {
+	/**
+	 * Returns an Array with all found Nodes.
+	 * Returns an Object with the first found Node.
```

Comment:
@nomego Let me know, I can re-add them but for the navigator I cannot see any use for them.